intermediate_source/torch_export_tutorial.py

```diff
@@ -0,0 +1,497 @@
+# -*- coding: utf-8 -*-
+
+"""
+torch.export Tutorial
+================
+**Author:** William Wen, Zhengxu Chen
+"""
+
+######################################################################
+# :func:`torch.export` is the PyTorch 2.0 way to export PyTorch models into
```

Comment:
also, you should say "trace-based" somewhere here -- it's really critical to understand that and right now it's dropped into the middle of a sentence about the FX graph format.